src/reference/informative/informC02_units_reference.rst

```diff
@@ -8,4 +8,155 @@
 
   .. container:: infospec
 
-    **TODO**
\ No newline at end of file
+    .. container:: heading3
+
+      Understanding units references
+
+    Units are different from other element types in that their names could refer either to other units which you've imported or created in your model, or to one of the list of :ref:`built-in units<table_built_in_units>`.  
```

Comment:
This isn't about units elements, it's about units references, which appear inside attributes!
I got confused reading this the first time, but think it's all down to the first sentence not really setting the scene